demo/config.js

```diff
@@ -177,301 +167,254 @@ shakaDemo.Config = class {
 
     addDRMAdvancedField(
-        MessageIds.VIDEO_ROBUSTNESS,
-        'videoRobustness',
-        robustnessSuggestions);
+        'Video Robustness', 'videoRobustness', robustnessSuggestions);
     addDRMAdvancedField(
-        MessageIds.AUDIO_ROBUSTNESS,
-        'audioRobustness',
-        robustnessSuggestions);
-    addDRMAdvancedField(
-        MessageIds.DRM_SESSION_TYPE,
-        'sessionType',
-        sessionTypeSuggestions);
+        'Audio Robustness', 'audioRobustness', robustnessSuggestions);
+    addDRMAdvancedField('Session Type', 'sessionType', sessionTypeSuggestions);
 
-    this.addRetrySection_('drm', MessageIds.DRM_RETRY_SECTION_HEADER);
+    this.addRetrySection_('drm', 'DRM Retry Parameters');
   }
 
   /** @private */
   addManifestSection_() {
-    const MessageIds = shakaDemo.MessageIds;
     const docLink = this.resolveExternLink_('.ManifestConfiguration');
-    this.addSection_(MessageIds.MANIFEST_SECTION_HEADER, docLink)
-        .addBoolInput_(MessageIds.IGNORE_DASH_DRM,
-            'manifest.dash.ignoreDrmInfo')
-        .addBoolInput_(MessageIds.AUTO_CORRECT_DASH_DRIFT,
+    this.addSection_('Manifest', docLink)
+        .addBoolInput_('Ignore DASH DRM Info', 'manifest.dash.ignoreDrmInfo')
+        .addBoolInput_('Auto-Correct DASH Drift',
             'manifest.dash.autoCorrectDrift')
-        .addBoolInput_(MessageIds.DISABLE_XLINK_PROCESSING,
+        .addBoolInput_('Disable Xlink processing',
             'manifest.dash.disableXlinkProcessing')
-        .addBoolInput_(MessageIds.XLINK_FAIL_GRACEFULLY,
+        .addBoolInput_('Xlink Should Fail Gracefully',
             'manifest.dash.xlinkFailGracefully')
-        .addBoolInput_(MessageIds.IGNORE_DASH_SUGGESTED_PRESENTATION_DELAY,
+        .addBoolInput_('Ignore DASH suggestedPresentationDelay',
             'manifest.dash.ignoreSuggestedPresentationDelay')
-        .addBoolInput_(MessageIds.IGNORE_DASH_EMPTY_ADAPTATION_SET,
+        .addBoolInput_('Ignore empty DASH AdaptationSets',
             'manifest.dash.ignoreEmptyAdaptationSet')
-        .addBoolInput_(MessageIds.IGNORE_DASH_MAX_SEGMENT_DURATION,
+        .addBoolInput_('Ignore DASH maxSegmentDuration',
             'manifest.dash.ignoreMaxSegmentDuration')
-        .addBoolInput_(MessageIds.IGNORE_HLS_TEXT_FAILURES,
+        .addBoolInput_('Ignore HLS Text Stream Failures',
             'manifest.hls.ignoreTextStreamFailures')
-        .addBoolInput_(MessageIds.IGNORE_HLS_IMAGE_FAILURES,
+        .addBoolInput_('Ignore HLS Image Stream Failures',
             'manifest.hls.ignoreImageStreamFailures')
-        .addTextInput_(MessageIds.DEFAULT_AUDIO_CODEC,
-            'manifest.hls.defaultAudioCodec')
-        .addTextInput_(MessageIds.DEFAULT_VIDEO_CODEC,
-            'manifest.hls.defaultVideoCodec')
-        .addBoolInput_(MessageIds.IGNORE_MANIFEST_PROGRAM_DATE_TIME,
+        .addTextInput_('Default Audio Codec', 'manifest.hls.defaultAudioCodec')
```

Comment:
Note for a low-priority follow-up: There are many configs specific to HLS or DASH that don't say so in the description here.  We could add those terms, but it may be even better to organize them into separate submenus.

Another low-priority follow-up: It occurs to me that it would be nice to have links to the docs of specific config fields from the config UI.